lib/internal/Magento/Framework/View/Template/Html/Minifier.php

```diff
@@ -131,8 +131,8 @@ public function minify($file)
                             '#(?<!:|\\\\|\'|")//(?!\s*\<\!\[)(?!\s*]]\>)[^\n\r]*#',
                             '',
                             preg_replace(
-                                '#(?<!:|\'|")//[^\n\r]*(\s\?\>)#',
-                                '$1',
+                                '#(?<!:|\'|")//[^\n\r]*(\?\>)#',
+                                ' $1',
```

Comment:
Why you did not simpy change the regular expression in `#(?<!:|'|")//[^\n\r]*(\s*\?\>)#`, instead of adding the prefixing space in the replacement?

Reply:
In case if the code will be like this one:
```
<?php//else:?>
```
if the replacement string will be just `'$1'`, we will get the result:
```
<?php?>
```
So it will cause calling of undefined conftant `php` and to prevent it I added the prefixing space in the replacement.